DFC.Microservices.Skills/docker-compose.yml

```diff
@@ -5,17 +5,4 @@ services:
     image: dfcmicroservicesskills
     build:
       context: .
-      dockerfile: DFC.Microservices.Skills/Dockerfile
-    depends_on:
-            - skillsmicroservicerepo
```

Comment:
Removed all of these as there are no way to populate this database where as Roy's code base has got node js project.
Managed to run his docker compose on the same network, so this service is talking to the database without having to merge it!!